lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -285,6 +337,33 @@ def find_module(self, fullname, path=None):
                 # call py2's internal loader
                 return pkgutil.ImpImporter(self._pathctx).find_module(fullname)
 
+    def find_spec(self, fullname, target=None):
+        split_name = fullname.split('.')
```

Comment:
This one is a bit more complicated it seems with the py2 branches, but I think we can still manage combine them.

Reply:
and yep.